MaibornWolff/codecharta
# Fix/1579/fixed folders map in delta mode causes errors

# {Fixed folders map in delta mode causes errors}

Closes: #1579 

## Description

  - Errors occurred in delta mode if names of root folders were different. Solved by:
    - showing aggregation of two projects if root folders have different names
    - regularly updating root data
 
## Screenshots or gifs
![image](https://user-images.githubusercontent.com/41293938/106925462-68c7e300-6710-11eb-961a-813e32c2b511.png)

## Test/Check scenario

  - Create two (or more) cc.json files and name the root folders differently
  - Load them into CodeCharta
  - Make sure there are no errors while switching modes and changing files on the file panel
